app/controllers/user_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
```

Comment:
- [ ] The controller, just like the routes, should use the plural form, instead of the singular.

  ```
  class UsersController < ApplicationController
  ```

- [ ] Kindly rename the file also from `user_controller.rb` to `users_controller.rb`.

Reply:
resolved